css-ui-4/Overview.bs

```diff
@@ -2358,6 +2358,19 @@ Switching appearance: the 'appearance' property</h3>
 					for defining which <a>widgets</a> have a <a>base appearance</a>.
 			</dl>
 
+     <dt><dfn>base-select</dfn>
+     <dd>
+       The effect of ''appearance/base-select'' depends on the element it is applied to:
+       <dl class=switch>
```

Comment:
I'd suggest two changes here:

* first, switch the order of the two items in this list so that the second one can just be "Everything else" (and you don't have to repeat "other than the `<select>` element in the definition either).
* second, for the `<select>`/`::picker(select)` case, I think it's better to fully defer to the definition of `appearance/base` (as you do for the other half) rather than trying to repeat it.

Reply:
Done and done, thanks!